client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -127,7 +145,9 @@ function PaymentMethodAdd(): ReactElement {
 			<DocumentHead title={ translate( 'Payment Methods' ) } />
 			<SidebarNavigation />
 
-			{ returnQueryArg && <AssignLicenseStepProgress currentStep="addPaymentMethod" /> }
+			{ ( !! returnQueryArg || product ) && (
```

Comment:
```suggestion
			{ ( !! returnQueryArg && !! product ) && (
```
I think we need both in order to consider this a part of the steps since no return = no flow / no next step and no product = no license issued.

Reply:
Not exactly, they happen in different situations, not always TRUE together, and for those cases we would want to have the progress bar being rendered.

1. User has previously issued a license, deleted the payment methods, went to assign that license and was sent to payment method input. In this case there is no `product` parameter, just `return`, it would need to show the progress bar, as they are seemingly resuming the process.
2. User is issuing a license. There is no `return` parameter as they will be sent to a next step as a regular issuing process, but there's the `product` parameter.

Reply:
Makes sense 👍